evidently/analyzers/test_utils.py

```diff
@@ -1,13 +1,12 @@
 import datetime
-import unittest
 
 import pandas
 
 from evidently.analyzers.utils import process_columns
 from evidently.pipeline.column_mapping import ColumnMapping
 
 
-class TestUtils(unittest.TestCase):
+class TestUtils:
```

Comment:
class can be removed, and all methods can be made just regular top level functions.

Reply:
I've moved the tests to functions, classes are removed for already moved to pyrest test